worker-sys/src/remote_service.rs

```diff
@@ -0,0 +1,18 @@
+use crate::Request as EdgeRequest;
+
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+extern "C" {
+    #[wasm_bindgen (extends = ::js_sys::Object, js_name = Fetcher)]
+    pub type RemoteService;
```

Comment:
We should try to either match the [workers-types](https://github.com/cloudflare/workers-types) name `Fetcher` or just use `Service`. I think we should replace all occurences of `RemoteService` with `Service`.